core/src/main/java/org/apache/gravitino/storage/relational/service/UserMetaService.java

```diff
@@ -151,10 +150,7 @@ public boolean deleteUser(NameIdentifier identifier) {
                 UserMetaMapper.class, mapper -> mapper.softDeleteUserMetaByUserId(userId)),
         () ->
             SessionUtils.doWithoutCommit(
-                UserRoleRelMapper.class, mapper -> mapper.softDeleteUserRoleRelByUserId(userId)),
-        () ->
-            SessionUtils.doWithoutCommit(
-                OwnerMetaMapper.class, mapper -> mapper.softDeleteOwnerRelByEntityId(userId)));
```

Comment:
Why do we need to delete this?

Reply:
User and group don't have ownership relation.